plans/benchmarks/benchmarks.go

```diff
@@ -260,15 +237,16 @@ func SubtreeBench(runenv *runtime.RunEnv) error {
 
 		for _, tst := range tests {
 			for i := 1; i <= iterations; i++ {
-				t := prometheus.NewTimer(tst.Summary)
+				t := time.Now()
 				_, err = client.Publish(ctx, tst.Topic, tst.Data)
 				if err != nil {
 					return err
 				}
-				t.ObserveDuration()
+				tst.Timer.UpdateSince(t)
+				runenv.R().RecordPoint(tst.Metric+ "_secs", time.Since(t).Seconds())
```

Comment:
When I run this and have a look at results.out, the results are hard to read, when converted to string. Maybe Seconds() could be changed to Milliseconds()  (uint64) rather than Seconds() (float64) 
```
96/results.out:{"ts":1588219276522873178,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000007843}}
96/results.out:{"ts":1588219276522885696,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000001202}}
96/results.out:{"ts":1588219276522895699,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":6.56e-7}}
96/results.out:{"ts":1588219276522935781,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000031799}}
96/results.out:{"ts":1588219276523233300,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000283827}}
96/results.out:{"ts":1588219276523249293,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000001009}}
96/results.out:{"ts":1588219276523258660,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":6.67e-7}}
96/results.out:{"ts":1588219276523275362,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000007412}}
96/results.out:{"ts":1588219276523287300,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":8.84e-7}}
96/results.out:{"ts":1588219276523297475,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":6.79e-7}}
96/results.out:{"ts":1588219276523314226,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":0.000007634}}
96/results.out:{"ts":1588219276523323765,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":8.03e-7}}
96/results.out:{"ts":1588219276523332504,"type":"point","name":"subtree_time_512_bytes_receive_secs","measures":{"value":7.09e-7}}
```
```